pyre/extra_tests/parity_tests/gc_varsize_item_const_shape.py

```
@@ -0,0 +1,79 @@
+The trigger was allocation layout rather than the loop bounds: the same body
+reading `N`/`WHICH` from `sys.argv` was clean, and wrapping the defaults to
+defeat constant folding did not change it.
+"""
+
+N = 15000
+WHICH = "head"
+
+ERR = ValueError("boom")
+
+
+def mid(i):
```

Ruff (0.16.2)

[warning] Unused function argument: `i`

(ARG001)